ownCloud.xcodeproj/project.pbxproj

```diff
@@ -5334,8 +5334,8 @@
 				APP_BUILD_FLAGS = "$(inherited)";
 				APP_BUILD_FLAGS_SWIFT = "$(APP_BUILD_FLAGS)";
 				APP_PRODUCT_NAME = ownCloud;
-				APP_SHORT_VERSION = 12.3.1;
-				APP_VERSION = 292;
+				APP_SHORT_VERSION = 12.4;
```

Comment:
not `12.4.0` ? @felix-schwarz

Reply:
On iOS it's not typical to include a patch version if that part is just a `.0`. Still added it with a new commit since few (if any) will notice it - and it's in line with the branch version number that way.